test/properties.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var jsc = require('jsverify');
+var R = require('ramda');
+
+exports.idempotent = function(f) {
+  return jsc.checkForall(useful, function(x) {
+    return R.equals(f(f(x)), f(x));
+  });
+};
+
+exports.involution = function(f) {
+  return jsc.checkForall(useful, function(x) {
+    return R.equals(f(f(x)), x);
+  });
+};
+
+
+var basic = jsc.sum([jsc.integer, jsc.string, jsc.bool, jsc.falsy]);
+
+var useful = jsc.sum([basic, jsc.array(basic), jsc.dict(basic), jsc.fn(basic)]);
```

Comment:
I realize there are conflicting styles, but I like to declare constants before declaring functions which reference them. If this seems reasonable to you, please move these definitions.

One other *very* minor stylistic issue: I like to leave two empty lines between imports and the rest of the code in a module. For example:

```javascript
'use strict';

const R = require('ramda');
const S = require('sanctuary');


const _ = R.__;

//  shout :: String -> String
exports.shout = S.compose(S.concat(_, '!'), S.toUpper);
```

I got this from [PEP 8][1], I believe.


[1]: https://www.python.org/dev/peps/pep-0008/